src/libs/fsdrv/lv_fs_posix.c

```diff
@@ -283,17 +283,17 @@ static lv_fs_res_t fs_dir_read(lv_fs_drv_t * drv, void * dir_p, char * fn)
     do {
         entry = readdir(dir_p);
         if(entry) {
-            if(entry->d_type == DT_DIR) lv_snprintf(fn, lv_strlen(entry->d_name) + 2, "/%s", entry->d_name);
-            else lv_strcpy(fn, entry->d_name);
+            if(entry->d_type == DT_DIR) lv_snprintf(fn, fn_len, "/%s", entry->d_name);
+            else lv_strncpy(fn, entry->d_name, fn_len);
         }
         else {
-            lv_strcpy(fn, "");
+            lv_strncpy(fn, "", fn_len);
         }
     } while(lv_strcmp(fn, "/.") == 0 || lv_strcmp(fn, "/..") == 0);
 #else
-    lv_strcpy(fn, next_fn);
+    lv_strcpy(fn, next_fn, fn_len);
```

Comment:
please revert this pr. There should use `lv_strncpy`

Reply:
don't to worry ever you push a new pr to fix this problem.

Reply:
Please also enable it in one of the CI build test.

Reply:
it's windows i don't know how to compile it

Reply:
https://github.com/lvgl/lvgl/pull/5364
PR is here